modules/node/src/setStateCommitment/setStateCommitment.repository.ts

```diff
@@ -1,36 +1,36 @@
 import { SetStateCommitmentJSON } from "@connext/types";
+import { deBigNumberifyJson, toBN } from "@connext/utils";
 import { EntityRepository, Repository } from "typeorm";
 
 import { SetStateCommitment } from "./setStateCommitment.entity";
 import { AppType } from "../appInstance/appInstance.entity";
+import { BigNumber } from "ethers/utils";
 
 export const setStateToJson = (entity: SetStateCommitment): SetStateCommitmentJSON => {
-  return {
+  return deBigNumberifyJson({
```

Comment:
Why are we making them BN and then deBignumberifying?

Reply:
the `SetStateCommitmentJSON` stores the `stateTimeout` and `versionNumbers` like:

```typescript
export type BigNumberJson = { _hex: HexString };
readonly stateTimeout: BigNumberJson;
readonly versionNumber: BigNumberJson;
```

The entities do not, and when I wrote this I was not aware of the `toBnJson` utils function

Will be fixed in #1037